canonical-websites/www.ubuntu.com
# 1602 containers lxd

## Done

* updated /containers/lxd to vbt
* added
```css
.p-list-step__content + p.p-list-step__content {
  margin-top: 1rem;
}
```
for code blocks in detailed stepped lists

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/containers/lxd)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/4.%20Containers?preview=containers-lxd.png) and the [demo](http://www.ubuntu.com-1602-containers-lxd.demo.haus/containers/lxd)

## Issue / Card

Fixes #1602
